app/views/content/help-and-advice.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Help and advice on getting into teaching"
+heading: "Help and advice"
+description: |-
+  Get help and advice on getting into teaching by talking to a dedicated adviser, going to a teacher training event, or registering for relevant email updates.
+navigation: 50
+image: "media/images/content/hero-images/0025.jpg"
+title_paragraph: We’re here to answer your questions and provide advice about teacher training, whether you’re just thinking about teaching or you’re ready to apply.
+calls_to_action:
+  chat:
+    name: chat
+    arguments: {}
+---
+
+## Attend an event
+
+Our events are a great way to meet teachers and other experts to talk about any concerns or questions you have about becoming a teacher.
+
+### Train to Teach events
+
+Train to Teach events are run by the Department for Education (DfE) and are free to attend. You'll be able to:
+
+- put your questions to expert advisers, teachers and training providers
+- chat with current teachers
+- find out what a career in teaching is really like
+- watch presentations which provide detailed guidance on how to get into teaching, the application process and funding your training
+
+### Online question and answer sessions
+
+There are online, text-based sessions where you can ask a panel of specialists the questions that matter to you. These sessions, run by DfE, cover a range of different topics so you can post specific questions and receive advice tailored to your particular circumstances.
+
+### School and university events
+
+You could also attend one of the School and University events, either online or in person, and hear directly from teacher training providers about the courses they offer and how to apply.
```

Comment:
```suggestion
You could also attend a school and university event, either online or in person, and hear directly from teacher training providers about the courses they offer and how to apply.
```

@EllieNodder Suggested change from the Events team for you to consider

NB once we have the new events design live, these events will be called 'Training provider events' rather than 'School and university events'